src/applications/disability-benefits/view-payments/components/view-payments-lists/helpers.js

```diff
@@ -159,3 +159,18 @@ export const reformatPaymentDates = payments => {
     };
   });
 };
+
+export const alertMessage = (
+  <va-alert close-btn-aria-label="Close notification" status="warning" visible>
+    <h2 slot="headline">We’re working to improve what we display here</h2>
```

Comment:
Really, really small nit. If we can make this an H3 and keep the same visual styling, that'd be preferable so it nests in the content flow. Otherwise, an H2 is valid markup and we can catch it as part of the work to make the table label a proper heading.